azblob/zt_url_container_test.go

```diff
@@ -221,8 +221,6 @@ func (s *aztestsSuite) TestContainerDeleteIfModifiedSinceTrue(c *chk.C) {
 	bsu := getBSU()
 	containerURL, _ := createNewContainer(c, bsu)
 
-	defer deleteContainer(c, containerURL)
```

Comment:
Why no delete now?

Reply:
The test is already deleting it, I'm really puzzled by why it was working previously.